### PR TITLE
lunasvg: add version 2.3.9

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **lunasvg/***

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/lunasvg/all/patches/2.3.9-0002-rename-inner-selector.patch [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+index 714228b..3ade738 100644
+--- a/source/parser.cpp
++++ b/source/parser.cpp
+@@ -1226,8 +1226,8 @@ bool RuleData::matchPseudoClassSelector(const PseudoClassSelector& selector, con
+         return element->parent == nullptr;
+ 
+     if(selector.type == PseudoClassSelector::Type::Is) {
+-        for(auto& selector : selector.subSelectors) {
+-            for(auto& sel : selector) {
++        for(auto& subselector : selector.subSelectors) {
```
Comment (uilianries) [inline recipes/lunasvg/all/patches/2.3.9-0002-rename-inner-selector.patch:11]: Hello @toge! Thank you for your pull request. Could you please explain the fix associated to this patch? It seems be a compilation error, probably it should be reported to the upstream as well.

Comment (toge) [reply recipes/lunasvg/all/patches/2.3.9-0002-rename-inner-selector.patch:11]: @uilianries 
It's a confusing code, but the name of this variable seems to be OK.
It has been already merged at https://github.com/sammycage/lunasvg/pull/147.

File: recipes/lunasvg/all/conandata.yml [hunk 2]
```diff
@@ -15,6 +18,13 @@ sources:
     url: "https://github.com/sammycage/lunasvg/archive/refs/tags/v2.3.1.tar.gz"
     sha256: "6492bf0f51982f5382f83f1a42f247bb1bbcbaef4a15963bbd53073cd4944a25"
 patches:
+  "2.3.9":
+    - patch_file: "patches/2.3.9-0001-fix-cmake.patch"
+      patch_description: "use external plutovg and fix installation path for conan"
+      patch_type: "conan"
+    - patch_file: "patches/2.3.9-0002-rename-inner-selector.patch"
+      patch_description: "rename inner selector to subselector"
+      patch_type: "portability"
```
Comment (uilianries) [inline recipes/lunasvg/all/conandata.yml:27]:
```suggestion
      patch_type: "portability"
      patch_source: "https://github.com/sammycage/lunasvg/pull/147"
```

List patch source

Comment (toge) [reply recipes/lunasvg/all/conandata.yml:27]: Sorry. I alread updated PR.

**Comment by toge** [comment]
@uilianries 
Cloud you review it again?

**Review by uilianries — APPROVED** [review]
LGTM. Thank you for your quick fix!

**Review by RubenRBS — APPROVED** [review]
Thanks!

**Pull request merged**